js/app.js

```diff
@@ -0,0 +1,152 @@
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['html', 'Bootstrap', 'JavaScript ES6'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid r',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Forth project',
+    title: 'Website Protfolio',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['Sass', 'Codekit', 'Terminal'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita,',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Fifth project',
+    title: 'Data Dashboard Healthcare',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['Sass', 'Bootstrap', 'JQuery'],
+    description:
+     'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita,',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Sixth project',
+    title: 'Website Protfolio',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['Terminal', 'Codepen', 'Github'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid repellat recusandae laboriosam natus culpa odit. ',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'last project',
+    title: 'Data Dashboard Healthcare',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['css', 'Codepen', 'Git & Github'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid repellat recusandae laboriosam natus culpa odit.',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+
+const divProject = document.createElement('div');
+
+// add modal function
+function addEvents() {
+  for (let i = 0; i < openPopupButtons.length; i += 1) {
+    openPopupButtons[i].addEventListener('click', (ev) => {
+      const projectId = ev.target.id;
+      divProject.classList.add('popup-overlay-objects');
+      document.querySelector('.popup-bg').style.display = 'flex';
+      // create techs
+      let techsHTML = '';
+      for (let k = 0; k < projects[i].technologies.length; k += 1) {
```

Comment:
- Kindly fix this error in chrome console: `` Cannot read property 'technologies' of undefined ``

Reply:
Done thanks for your review